CHANGELOG.md

```diff
@@ -50,13 +55,32 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Fix initial scans tests ([#5032](https://github.com/wazuh/wazuh-qa/pull/5032)) \- (Framework + Tests)
+- Handle VDT data missing in wazuh-db API ([#5014](https://github.com/wazuh/wazuh-qa/pull/5014)) \- (Framework + Tests)
+- Fixed x-axis labels in data-visualizer script ([#4987](https://github.com/wazuh/wazuh-qa/pull/4987)) \- (Framework)
+- Fix monitoring module for e2e tests ([#4959](https://github.com/wazuh/wazuh-qa/pull/4959)) \- (Framework)
```

Comment:
Dupe from line 66

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5063/commits/9c4fc1ab3d533bb1469a490401090672f142f6ac